src/ServiceControl.Audit.Persistence/PersistenceManifest.cs

```diff
@@ -78,28 +78,16 @@ static string GetAssemblyDirectory()
             return Path.GetDirectoryName(assemblyLocation);
         }
 
-        public static string Find(string persistenceType)
-        {
-            if (persistenceType == null)
-            {
-                throw new Exception("No persistenceType has been configured. Either provide a Type or Name in the PersistenceType setting.");
-            }
-
-            var persistenceManifestDefinition = PersistenceManifests.FirstOrDefault(w => w.IsMatch(persistenceType));
-
-            return persistenceManifestDefinition?.TypeName ?? persistenceType;
-        }
-
-        public static string GetPersistenceFolder(string persistenceType)
+        public static PersistenceManifest Find(string persistenceType)
         {
             if (persistenceType == null)
             {
                 throw new Exception("No persistenceType has been configured. Either provide a Type or Name in the PersistenceType setting.");
             }
 
-            var persistenceManifestDefinition = PersistenceManifests.FirstOrDefault(w => w.IsMatch(persistenceType));
+            var persistenceManifest = PersistenceManifests.FirstOrDefault(w => w.IsMatch(persistenceType));
```

Comment:
```suggestion
            return PersistenceManifests.FirstOrDefault(w => w.IsMatch(persistenceType));
```

Reply:
Took a look, and would prefer not to make this change since there are two `PersistenceManifest` classes and a `TransportManifest` class that all have the same basic code shape. I wouldn't want to change `TransportManifest` in a similar way, so I'd prefer to keep them all the same.

Also, having the local there does make it easier to see the value when debugging, so in general I don't mind having it there.